packages/react/src/components/accordion/Accordion.tsx

```diff
@@ -113,24 +132,30 @@ export const Accordion = ({
       style={style}
       id={accordionId}
     >
-      <div className={styles.accordionHeader}>
-        <div role="heading" aria-level={headingLevel} id={`${accordionId}-heading`}>
-          {heading}
-        </div>
-        <button
-          type="button"
-          {...buttonProps}
-          className={styles.accordionButton}
+      <div className={classNames(styles.accordionHeader)}>
+        <div
```

Comment:
I'd guess this is not a button but div because of heading etc.?

Reply:
Good catch. We could apply directly the button tag here. Though tested it and it would require some more work since user agents of the browsers will add some styles based on the button tag. Maybe we go with this for simplicity for now.